versions.config

```diff
@@ -4,11 +4,11 @@ ROOTFS_TAR_CHECKSUM="d1e7e6d48a25b4a206c5df99ecb8815388ec6945e4f97e78413d5a80778
 
 # name of the ready made raw image for RPi
 RAW_IMAGE="rpi-raw.img"
-RAW_IMAGE_VERSION="v0.2.2"
-RAW_IMAGE_CHECKSUM="2fbeb13b7b0f2308dbd0d82780b54c33003ad43d145ff08498b25fb8bbe1c2c6"
+RAW_IMAGE_VERSION="master"
```

Comment:
I tried reverting this file but seem not to be able to???

Reply:
how so "not able to" is it a permissions issue?

Reply:
If you are finding issues building, it's a good idea to check the upstream project https://github.com/hypriot/image-builder-rpi/blob/master/versions.config - as Raspbian is a moving target that gets updated and requires tweaking the build.